ui/app/[locale]/home/page.tsx

```diff
@@ -297,7 +297,14 @@ async function ConnectedClustersTable({
   const allClusters = await clusterPromise;
   const clusters = allClusters.map<EnrichedClusterList>((c) => {
     async function getNodesCounts() {
-      const cluster = await getKafkaCluster(c.id);
+      let cluster;
+
+      if (c.meta.configured === true) {
+        cluster = await getKafkaCluster(c.id);
+      } else {
+        cluster = null;
+      }
```

Comment:
I think we want this in the getCluster, so to protect the pages that depend on it and properly redirect users to the homepage

Reply:
I see what you mean. Since it's called from the homepage itself, where does the redirect happen?

Reply:
It doesn't, it's handled on a per-route basis checking for a null return value.

Reply:
In that case, maybe getKafkaCluster just needs to handle the 404 error. It won't have access to the meta information from the cluster list entry (without an additional request). Wdyt?

Reply:
Returning null on 404 from `getKafkaCluster` results in a page error, mostly from other calls for consumer groups, etc. failing. I think the only way to get to those pages would be to obtain the cluster Id indirectly and use it in the URL, correct?

Reply:
Right my comment was to cover the case of users typing the url by hand, or getting one shared, or from a bookmark

Reply:
It's fine to keep it this way for the moment